app/assets/javascripts/govuk_publishing_components/analytics/track-click.js

```diff
@@ -30,6 +30,8 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   }
 
   GemTrackClick.prototype.handleClick = function (target) {
+    console.log('HELLO')
+
```

Comment:
Looks like there's a stray console.log message here to be removed.

Reply:
ah thanks, will remove that